### DISCUSSION TITLE
WebEx

### DISCUSSION THREAD

**Discussion by Greymourn** [tom-englert/RegionToShare#29] [General]
Thank you for developing this, but I seem to have issues getting this to work on WebEx. It works fine in MS Teams. I have the 49" Samsung and it still makes the screen share look like it's still trying to display two screens. Any ideas on what I might need to do to fix?

**Comment by tom-englert** [comment]
Just updated the Readme:

This tool simply mirrors the content of a screen region into a hidden window. In your meeting app you then can just share the content of this hidden window. 

**Region to Share is not aware of your meeting app nor what the meeting app is doing with the content of the window.** 
It's up to your meeting app whether it properly shares this hidden windows content or not - there is nothing Region to Share can do about this.
